DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -63,6 +69,8 @@ RemovalMethod removalMethod
 
         public CentreCourseDetails GetCentreCourseDetails(int centreId, int? categoryId);
 
+        public CentreCourseDetails GetDelegateCourseDetails(int centreId, int? categoryId);

```

Comment:
I don't think the naming is very good for this method. It implies to me that it has something to do with getting a course for a delegate, but it doesn't.